db/schema/patch-007-008.sql

```diff
@@ -0,0 +1,175 @@
+-- Add a lockedAt column for account locking.
+
+ALTER TABLE accounts ADD COLUMN lockedAt BIGINT UNSIGNED DEFAULT NULL;
+
+-- the codes to unlock an account
+CREATE TABLE IF NOT EXISTS accountUnlockCodes (
+  uid BINARY(16) PRIMARY KEY NOT NULL,
+  unlockCode BINARY(16) NOT NULL,
+  FOREIGN KEY (uid)
+      REFERENCES accounts(uid)
+      ON DELETE CASCADE
```

Comment:
@rfk - I have removed the FK constraint as well as updated `deleteAccount` to remove any unlockCodes. should `resetAccount` be updated to remove unlockCodes as well? Where is `resetAccount` called from?